lucidworks/connectors-sdk-resources
# CONN-2332: bump up gradle to 7

# Tl;dr

Bump up gradle to 7 and update build configuration.

# Details
https://stackoverflow.com/questions/47910578/not-able-to-copy-configurations-dependencies-after-upgrading-gradle-plugin-for-a

# How to verify

Plugin zips should be built when running:
```
./gradlew clean assemblePlugins 
```
Upload one of the into Fusion to verify it is operational
